packages/docusaurus-utils/src/gitUtils.ts

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import path from 'path';
+import shell from 'shelljs';
+
+export const getCommitterDateForFile = (
```

Comment:
Intuitively, I don't understand what "commiterDate" is 🤷‍♂️  maybe it's not a good name or at least it requires some additional comments?